Formula/dscanner.rb

```diff
@@ -0,0 +1,27 @@
+class Dscanner < Formula
+  desc "The Dscanner untility for the D programming language"
```

Comment:
This doesn’t really tell us what this utility is for. Could you please write a more descriptive description?